configure.ac

```diff
@@ -62,6 +62,7 @@ AC_USE_SYSTEM_EXTENSIONS
 
 AC_PROG_CXX
 AC_LANG_COMPILER(C++)
+AC_PROG_CPP
```

Comment:
OK. Is this where it belongs though, it seems the CXX and C++ bits should be together without CPP in the middle

Reply:
I'll have to run through it again, but the order for invoking the macros is a bit fragile (autoconf/automake whine a lot about not calling things in the right order).